posthog/tasks/calculate_event_property_usage.py

```diff
@@ -259,16 +261,23 @@ def _get_event_properties(team_id: int, since: timezone.datetime) -> List[Tuple[
     return sync_execute(GET_EVENT_PROPERTIES, {"team_id": team_id, "timestamp": since})
 
 
-def _get_insight_query_usage(team_id: int, since: datetime) -> Tuple[List[str], Counter[PropertyIdentifier]]:
+def _get_insight_query_usage(team_id: int, since: datetime) -> Tuple[List[str], TCounter[PropertyIdentifier]]:
     event_usage: List[str] = []
-    counted_properties: Counter[PropertyIdentifier] = Counter()
+    counted_properties: TCounter[PropertyIdentifier] = Counter[PropertyIdentifier]()
```

Comment:
I don't think this would work?

```suggestion
    counted_properties: TCounter[PropertyIdentifier] = Counter()
```